src/index.js

```diff
@@ -123,14 +123,18 @@ class HCaptcha extends React.Component {
       const { isApiReady } = this.state;
       if (!isApiReady) return;
 
+      const { sitekey, size, theme, tabindex, languageOverride, endpoint } = this.props;
```

Comment:
Is it a complete list of valid properties?

Reply:
@wontem `sitekey`, `size`, `theme` and `hl` (`languageOverride`) can be found in v1 docs, I didn't find `tabindex` in docs but seems that it's used. Will remove `endpoint` from this list since it's not needed here, thanks, just no linter setup here to hint :)

Reply:
@wontem cleaned up [here](https://github.com/hCaptcha/react-hcaptcha/pull/106/commits/44ac6e6dfc9fd857791602f06228d15e5b31fe13)